core/server/common/src/test/java/alluxio/master/journal/raft/SnapshotReplicationManagerTest.java

```diff
@@ -120,7 +120,7 @@ public void before() throws Exception {
 
   private SimpleStateMachineStorage getSimpleStateMachineStorage() throws IOException {
     RaftStorage rs = new RaftStorageImpl(mFolder.newFolder(CommonUtils.randomAlphaNumString(6)),
-            RaftServerConfigKeys.Log.CorruptionPolicy.getDefault());
+            RaftServerConfigKeys.Log.CorruptionPolicy.getDefault(), 0);
```

Comment:
why do we need to change this raft storage when we upgrade the ozone version?

Reply:
the ratis has a non-backward compatible change for version 2.2 reference https://github.com/apache/ratis/pull/565 can get more context